user/pages/05.blog/2017-08-14-statusbox/item.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Statusbox
+date: 18:45 2017-08-14
+author: Marcel Fuhrmann
+body_classes: header-lite fullwidth blogstyling
+taxonomy:
+    category: news
+    tag: [startpage, graphs, overview]
+---
+
+Hey folks,
+
+maybe you still remember the discussion about the [Statusbox feature](https://wiki.opennms.org/wiki/DevProjects/Status_Box) which Markus von Rüden started at the [Easterhack](https://www.opennms.org/en/blog/2017-05-11-easterhack-review) session.
+At that time I already liked the idea and to be honest, I was a little bit disappointed that it wasn't possible to complete it on this weekend.
+After some days I read Markus' comment in his [pull request](https://github.com/OpenNMS/opennms/pull/1483)...
+
+“As this feature was intended to be a small enhancement but a lot of issues were raised while developing this issue, I am now going to burry it.”
+... and I was really sad to hear that. Certainly I could understand it.
+
+But good things come to those who wait! The OpenNMS guys revived the Statusbox feature at the DevJam in Montreal and they followed through to complete it!
+But to cut a long story short...
+
+![donut](donut.gif)
+
+In my opinion this new feature is one of the most handy features since several updates - from the point of an OpenNMS user who works everyday with ONMS. It improves the usability and brings a new fancy look at the starting page.
```

Comment:
Do we follow the one sentence per line rule here as well? If so it is not followed (-:

In addition I wouldn't use ONMS but OpenNMS instead.